### PR TITLE
Fix travis clang

### PR DESCRIPTION
**Pull request by richsalz**

### CODE REVIEW

**Comment by ekasper** [comment]
ubsan builds are passing. Try switching to the old clang for asan; that seems to work.

**Comment by richsalz** [comment]
okay @ekasper, pushed a commit that uses 'clang' for asan; let's see what happens.

**Comment by richsalz** [comment]
It builds.  Can some else review?  Would be nice to get our builds green agai.

File: .travis.yml [hunk 1]
```diff
@@ -32,15 +34,19 @@ matrix:
...
+#          compiler: clang-3.6
+#          compiler: clang
+#          env: CONFIG_OPTS="no-shared enable-asan"
+#        - os: linux
+#          compiler: clang-3.6
+#          env: CONFIG_OPTS="no-asm enable-ubsan enable-rc5 enable-md2 -fno-sanitize=alignment"
         - os: linux
           compiler: gcc-5
-          env: CONFIG_OPTS="no-asm enable-ubsan enable-rc5 enable-md2 -fno-sanitize=alignment"
+          env: CONFIG_OPTS="no-shared no-asm enable-rc5 enable-md2"
```
Comment (levitte) [inline .travis.yml]: `no-shared`, why?

Comment (levitte) [inline .travis.yml]: Oh, never mind, I think I see why...

**Comment by levitte** [comment]
Ship

**Pull request closed without merging**